libs/ui/elements/src/lib/notification/notification.component.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+} from '@angular/core'
+import { CommonModule } from '@angular/common'
+import { MatIconModule } from '@angular/material/icon'
+import { ButtonComponent } from '@geonetwork-ui/ui/inputs'
+
+@Component({
+  selector: 'gn-ui-notification',
+  standalone: true,
+  imports: [CommonModule, MatIconModule, ButtonComponent],
+  templateUrl: './notification.component.html',
+  styleUrls: ['./notification.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NotificationComponent {
+  @Input() type: 'info' | 'warning' | 'error' | 'success' = 'info'
+  @Input() title: string
+  @Input() text: string
+  @Input() closeMessage?: string
+  @Output() notificationClose = new EventEmitter<void>()
+
+  handleClose(event?: Event) {
+    event?.preventDefault()
```

Comment:
Out of curiosity, what do you need to prevent here?

Reply:
if the "close" is done by clicking on the close message, since this is a link we need to prevent the browser from actually following the link
A better solution would be to have buttons that can look like links (and thuse have no default behavior), but I didn't go that far